### PR TITLE
1.2.0

### PR DESCRIPTION
**Pull request by jowong4**
This PR contains misc changes to master before 1.2.0 release. Changes include:

- Updating README
- Moving catch.hpp to vendor and undo white space changes
- add vendor to include paths

With regards to "add vendor to include paths", we think that it makes more sense for the user to just do `-I/path/btl_bloomfilter/` instead of `-I/path/btl_bloomfilter/ -I/path/btl_bloomfilter/vendor`

Once there are no more changes, I will do a final commit where I change `configure.ac`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,26 +10,25 @@ Fast Bloom filter loading using the rolling hash function.
 #include "BloomFilter.hpp"
 #include <vector>
 #include <string>
-#include "ntHashIterator.hpp"
+#include "vendor/ntHashIterator.hpp"
```
Comment (sjackman) [inline README.md]: Should `BloomFilter.hpp` include `ntHashIterator.hpp` for the user?

Comment (jowong4) [reply README.md]: I imagine the original intent was to let the user use whatever hashing algorithm he wants to, and if he is using the bloom filter data structure but not using nucleotide sequences, I don't think nthash is suitable anymore.

**Comment by jowong4** [comment]
So I realized the AdHoc tests weren't running so I added them back to check. I also resolved errors as a results of using an older in the AdHoc tests. If everything is fine, my next commit will be the `configure.ac` change

**Comment by jowong4** [comment]
@JustinChu @sjackman 

Under "acknowledgements" should "rolling hash implementation by Hamid Mohamadi" be " nthash implementation by Hamid Mohamadi" instead and then a link to his repo or nthash containing a hyperlink?

**Comment by JustinChu** [comment]
Yeah, and adding a hyperlink makes a lot of sense.

**Comment by jowong4** [comment]
Keeping this branch for now

**Pull request merged**